packages/sn-document-viewer-react/src/components/DocumentViewerLayout.tsx

```diff
@@ -169,6 +229,33 @@ export class DocumentViewerLayoutComponent extends React.Component<
             padding={8}
             activePage={this.state.activePage}
           />
+          <Drawer
+            variant={'persistent'}
+            open={this.props.showComments}
+            anchor="right"
+            SlideProps={this.props.drawerSlideProps}
+            PaperProps={{
+              style: {
+                position: 'relative',
+                width: this.props.showComments ? '340px' : 0,
+                height: '100%',
+                overflow: 'hidden',
+              },
+            }}>
+            <CommentsContainer>
+              <Typography variant="h4">{this.props.localization.commentSideBarTitle}</Typography>
+              <CreateComment
```

Comment:
A feedback on UX:
 - I don't have an option to *cancel* adding a comment (e.g. clicking away or a "Cancel" button)
 - The "Create comment" component always appears *below* the first comment but my new comment will be ordered based on the tick's location. Once I press submit, I've lost the context 😿  Maybe it would be nice to edit comments 'inline'.